install/windows-10.md

```diff
@@ -9,16 +9,16 @@ __Bitte die folgenden Punkte unbedingt lesen:__
 + Diese Anleitung mag relativ lang erscheinen und im ersten Moment abschreckend wirken.
 Wir versichern dir, dass diese Länge aber viel mehr ein Zeichen von Ausführlichkeit
 ist, um sicherzustellen, dass du auch folgen kannst, wenn du bisher keine Berührungen
-mit den Inhalten des Toolbox Workshops hattest. 
+mit den Inhalten des Toolbox Workshops hattest.
 
 + Befolge die Anleitung daher konzentriert und gründlich, um keinen Schritt zu vergessen
 und kontrolliere jedes Mal, dass du die Befehle richtig kopiert oder abgetippt hast.
 
 + Alle drei Abschnitte (Installieren, Testen, Aktualisieren) und deren Unterabschnitte sind notwendig.
-Den Beginn jedes dieser drei Abschnitte erreichst du über die drei Buttons unter der Einleitung. 
+Den Beginn jedes dieser drei Abschnitte erreichst du über die drei Buttons unter der Einleitung.
```

Comment:
These buttons don't exist any longer, as we now have the TOC. We should delete this point